kubernetes/helm_charts/core/assessment/values.j2

```diff
@@ -0,0 +1,49 @@
+#jinja2:lstrip_blocks: True
+
+### Default variable file for assessment-service ###
+
+namespace: {{ namespace }}
+imagepullsecrets: {{ imagepullsecrets }}
+dockerhub: {{ dockerhub }}
+
+replicaCount: {{assessment_replicacount|default(1)}}
+repository: {{assessment_repository|default('assessment-service')}}
+image_tag: {{ image_tag }}
+resources:
+  requests:
+    cpu: {{assessment_cpu_req|default('50m')}}
+    memory: {{assessment_mem_req|default('50Mi')}}
+  limits:
+    cpu: {{assessment_cpu_limit|default('1')}}
+    memory: {{assessment_mem_limit|default('500Mi')}}
+network:
+  port: 9000
+strategy:
+  type: RollingUpdate
+  maxsurge: 1
```

Comment:
Remove max.*. We are not using these in deployment manifests.

Reply:
But If we don't have this will kube wait for the container to be up, before killing the old one ?